x-pack/plugins/enterprise_search/public/applications/app_search/components/schema/components/empty_state.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiPanel, EuiEmptyPrompt, EuiButton } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { DOCS_PREFIX } from '../../../routes';
+
+export const EmptyState: React.FC = () => {
+  return (
+    <EuiPanel color="subdued">
+      <EuiEmptyPrompt
+        iconType="gear"
+        title={
+          <h2>
+            {i18n.translate('xpack.enterpriseSearch.appSearch.engine.schema.empty.title', {
+              defaultMessage: 'Create a schema',
+            })}
+          </h2>
+        }
+        body={
+          <p>
+            {i18n.translate('xpack.enterpriseSearch.appSearch.engine.schema.empty.description', {
+              defaultMessage:
+                'Create schema fields in advance, or index some documents and a schema will be created for you.',
+            })}
+          </p>
+        }
+        actions={
+          <EuiButton
+            size="s"
+            target="_blank"
+            iconType="popout"
+            href={`${DOCS_PREFIX}/indexing-documents-guide.html#indexing-documents-guide-schema`}
```

Comment:
@daveyholler / @chriscressman  Does this documentation link look right to you? 

https://www.elastic.co/guide/en/app-search/master/indexing-documents-guide.html#indexing-documents-guide-schema

It looks like there isn't a specific guide/docs page for Schema.

Feel free to wordsmith the copy here also, I made it up somewhat!

Reply:
Sounds great!! Definitely let us know when documentation changes go out - currently almost all empty states are linking out to docs now (👩‍🎓) so we'll definitely want to keep that up to date.